src/NServiceBus.AcceptanceTests/NServiceBus.AcceptanceTests.csproj

```diff
@@ -252,7 +252,7 @@
     <Compile Include="Serialization\When_wrapping_is_not_skipped.cs" />
     <Compile Include="Serialization\When_xml_serializer_used_with_unobtrusive_mode.cs" />
     <Compile Include="Routing\NativePublishSubscribe\When_multi_subscribing_to_a_polymorphic_event.cs" />
-    <Compile Include="Routing\MessageDrivenSubscriptions\When_using_autosubscribe_with_missing_routing_information.cs" />
+    <Compile Include="Routing\MessageDrivenSubscriptions\When_autosubscribe_with_missing_routing_information.cs" />
```

Comment:
Any concerns with this file being named differently than it is in master and develop? Its `When_autosubscribe_with_missing_publisher_information.cs` in those locations.

Reply:
Not sure, I just made it shorter to get it compiling

Reply:
Could be that we renamed btw v6 and v7